store/sdk/src/main/java/org/apache/carbondata/sdk/file/CarbonReader.java

```diff
@@ -90,6 +93,20 @@ public T readNextRow() throws IOException, InterruptedException {
     return currentReader.getCurrentValue();
   }
 
+  /**
+   * Read and return next batch row objects
+   */
+  public Object[] readNextBatchRow() throws Exception {
+    validateReader();
+    int batch = Integer.parseInt(CarbonProperties.getInstance()
```

Comment:
What is this property is not set ? we get NPE here

Reply:
ok, I added default batch size

Reply:
no need batch in here, I removed.